android/app-example/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".app.RootActivity">
+        <activity android:name=".app.RecyclerViewTestActivity">
```

Comment:
> To test in example app, set it as launcher in app-example / AndroidManifest.xml

Is this change intended? Or you just forgot to switch it back to RootActivity?

Reply:
Yes, forgot. Thanks!